lib/transaction.dart

```diff
@@ -0,0 +1,24 @@
+class Transaction {
+  /// RevenueCat Id associated to the transaction.
+  final String revenuecatId;
```

Comment:
I just realized the c is not capitalized in the Android and iOS class, and that the d is lowercase in `Id` 🤦

Reply:
I'm capitalizing the C, but the d in `ID` I'm not updating right now because it should match the original iOS code - once we update it in `purchases-ios`, we can update it here

Reply:
actually, it's a little more problematic - it's capitalized here: 
https://github.com/RevenueCat/purchases-hybrid-common/blob/develop/ios/PurchasesHybridCommon/PurchasesHybridCommon/RCTransaction%2BHybridAdditions.m#L17
and not here: 
https://github.com/RevenueCat/purchases-hybrid-common/blob/develop/android/src/main/java/com/revenuecat/purchases/common/mappers/TransactionMapper.kt#L7
so we'll need another patch release to fix it, otherwise mappers won't work. 
I'll take care of it

Reply:
okay, tested and working with common 1.4.2 https://github.com/RevenueCat/purchases-hybrid-common/pull/55

This PR will need to bump the version number once the hybrid-common release is out, though. adding a hold until then.